### PR TITLE
fix: fix test deployment workflow file

### PR DESCRIPTION
**Pull request by MSzalowski**
## List of changes

- fix test deployment workflow file

### CODE REVIEW

**Review by pmbinapps — APPROVED** [review]

File: .github/workflows/build-and-deploy-dev.yml [hunk 1]
```diff
@@ -23,8 +23,8 @@ jobs:
       GA_CLIENT_EMAIL: ${{ secrets.GA_CLIENT_EMAIL }}
       GA_PRIVATE_KEY: ${{ secrets.GA_PRIVATE_KEY }}
       GRAFANA_ADMIN_PASSWORD: ${{ secrets.GRAFANA_ADMIN_PASSWORD }}
-      GRAFANA_SLACK_OAUTH_TOKEN: ${{ secrets.GRAFANA_SLACK_OAUTH_TOKEN }}
       GRAFANA_SLACK_RECIPIENT: ${{ secrets.GRAFANA_SLACK_RECIPIENT }}
+      GRAFANA_SLACK_OAUTH_TOKEN: ${{ secrets.GRAFANA_SLACK_OAUTH_TOKEN }}
```
Comment (placek) [inline .github/workflows/build-and-deploy-dev.yml:27]: why the change of order?

File: .github/workflows/build-and-deploy-test.yml [hunk 2]
```diff
@@ -32,7 +31,9 @@ jobs:
       TEST_NGINX_BASIC_AUTH: ${{ secrets.TEST_NGINX_BASIC_AUTH }}
       PIPELINE_URL: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
       SENTRY_DSN: ${{ secrets.SENTRY_DSN_FRONTEND }}
-      PIPELINE_URL: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
```
Comment (placek) [inline .github/workflows/build-and-deploy-test.yml:35]: This is used in info target [here](https://github.com/IntersectMBO/govtool/blob/develop/scripts/govtool/info.mk#L2).
Without that you'll have no link to the pipeline in slack notification.

File: .github/workflows/build-and-deploy-test.yml [hunk 3]
```diff
@@ -23,7 +23,6 @@ jobs:
       GRAFANA_ADMIN_PASSWORD: ${{ secrets.GRAFANA_ADMIN_PASSWORD }}
       GRAFANA_SLACK_RECIPIENT: ${{ secrets.GRAFANA_SLACK_RECIPIENT }}
       GRAFANA_SLACK_OAUTH_TOKEN: ${{ secrets.GRAFANA_SLACK_OAUTH_TOKEN }}
-      SENTRY_DSN_BACKEND: ${{ secrets.SENTRY_DSN_BACKEND }}
```
Comment (placek) [inline .github/workflows/build-and-deploy-test.yml:26]: again - why reorder?

**Review by placek — COMMENTED** [review]
Apart from [this part](https://github.com/IntersectMBO/govtool/pull/1025/files#diff-d63278f79551d70a0ecc23a55e3dc2e84fbd33ca883e2eed36768ae27c245fa5R35-R36) I find this PR strange…
